src/audio/Audio.js

```diff
@@ -106,15 +110,15 @@ Audio.prototype = Object.assign( Object.create( Object3D.prototype ), {
 
 		}
 
-		var source = this.context.createBufferSource();
+		this._startedAt = this.context.currentTime + startTime;
 
+		var source = this.context.createBufferSource();
 		source.buffer = this.buffer;
 		source.loop = this.loop;
 		source.loopStart = this.loopStart;
 		source.loopEnd = this.loopEnd;
 		source.onended = this.onEnded.bind( this );
-		this.startTime = this.context.currentTime;
-		source.start( this.startTime, this.offset, this.duration );
+		source.start( this._startedAt, this._pausedAt + this.offset, this.duration );
```

Comment:
Do we still need `this.offset` in this line?

Reply:
I think so.

`this.offset` is relative to the sample. `this.pausedAt` is relative to `context.currentTime`.

Say that you have a 20 seconds sound...

```js
var audio = new THREE.Audio();
audio.offset = 10;
audio.play(); // starts from second 10
// wait 3 seconds
audio.pause(); // this._pausedAt is 3
audio.play(); // should continue where it left of (10 + 3)
```

I think it would be more clear to abstract all this with something like...

```js
audio.range = [ 10, 20 ]; 
```

But, we can revisit this in the future.

Reply:
Okay, makes sense!